docs/spec/staking/README.md

```diff
@@ -2,30 +2,38 @@
 
 ## Abstract
 
-This paper specifies the Staking module of the Cosmos-SDK, which was first described in the [Cosmos Whitepaper](https://cosmos.network/about/whitepaper) in June 2016. 
+This paper specifies the Staking module of the Cosmos-SDK, which was first
+described in the [Cosmos Whitepaper](https://cosmos.network/about/whitepaper)
+in June 2016. 
 
-The module enables Cosmos-SDK based blockchain to support an advanced Proof-of-Stake system. In this system, holders of the native staking token of the chain can become candidate validators and can delegate tokens to candidate validators, ultimately determining the effective validator set for the system.
+The module enables Cosmos-SDK based blockchain to support an advanced
+Proof-of-Stake system. In this system, holders of the native staking token of
+the chain can become validators and can delegate tokens to validator
+validators, ultimately determining the effective validator set for the system.
 
-This module will be used in the Cosmos Hub, the first Hub in the Cosmos network.
+This module will be used in the Cosmos Hub, the first Hub in the Cosmos
+network.
 
 ## Contents
 
-The following specification uses *Atom* as the native staking token. The module can be adapted to any Proof-Of-Stake blockchain by replacing *Atom* with the native staking token of the chain.
+The following specification uses *Atom* as the native staking token. The module
+can be adapted to any Proof-Of-Stake blockchain by replacing *Atom* with the
```

Comment:
"*proof-of-stake*"

Staking token must be scarce.

Reply:
addressed